src/scriptworker/task.py

```diff
@@ -51,6 +52,39 @@ def worst_level(level1, level2):
     return level1 if level1 > level2 else level2
 
 
+# get_task_definition {{{1
+async def get_task_definition(queue, task_id, exception=TaskclusterFailure):
+    """Get the task definition from the queue.
+
+    Detect whether the task definition is empty, per bug 1618731.
+
+    Args:
+        queue (taskcluster.aio.Queue): the taskcluster Queue object
+        task_id (str): the taskId of the task
+        exception (Exception, optional): the exception to raise if unsuccessful.
```

Comment:
Question: What is the pro of making `exception` configurable?

Reply:
I wanted to use `CoTError` in `cot.verify`, but that exception doesn't necessarily make sense if we call this function elsewhere in the future.